recipes/pydoe3/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2014, Abraham D. Lee
```

Comment:
The license is in the tarball. You can remove this file.

Reply:
ok